Nomenclature (#1170)
OctopusDeploy/docs: docs/getting-started/terminology.md

---
title: Terminology
description: A page containing the definitions of terms used in Octopus Deploy.
position: 10
---

As you start using Octopus Deploy, you are going to learn a lot of new terms.  Below is a list of common terms used through the product and documentation.

> - **Octopus Server**: the service responsible for hosting the Octopus UI and orchestrating deployments.

Reviewer: ```suggestion
- **Octopus Server**: the service responsible for hosting the Octopus Web Portal, Rest API, and orchestrating deployments.
```

- **Self-Hosted**: when you install the Octopus Server on your infrastructure.
- [**Octopus Cloud**](/docs/octopus-cloud/index.md): the hosted version of Octopus Server; but you still install tentacles on your infrastructure that connect to Octopus Cloud.
- **Task**: a unit of work performed by the Octopus Server.  A task can be a deployment, a machine health check, a runbook run, and more.  All tasks are dropped onto the task queue and picked up in a FIFO order (unless the task is scheduled to run at a specific time).
- [**Task Cap**](/docs/support/increase-the-octopus-server-task-cap.md): how many concurrent tasks the Octopus Server can process.  For self-hosted instances this can be increased from the default of 5.
- **Instance**: the database, file share, and 1 to N nodes running the Octopus Server service.  Each self-hosted Octopus Deploy license allows for three active instances.
- [**Node**](/docs/administration/high-availability/managing-high-availability-nodes.md): an individual server running the Octopus Server in an Octopus Instance.  

> - [**Infrastructure**](/docs/infrastructure/index.md): what the Octopus Server will deploy to.

Reviewer: ```suggestion
## Infrastructure

 Your [**Infrastructure**](/docs/infrastructure/index.md) is servers, services, and accounts where the Octopus Server will deploy your software.
```

    - [**Worker Pools**](/docs/infrastructure/workers/worker-pools.md): a group of workers.  One pool might be in a particular network security zone. Another pool might have a specific set of tools installed.
    - [**Health Check**](/docs/infrastructure/deployment-targets/machine-policies.md#health-check): a task Octopus periodically runs on deployment targets and workers to ensure that they are available.
    - [**Machine Policies**](/docs/infrastructure/deployment-targets/machine-policies.md): groups of settings that can be applied to Tentacle and SSH endpoints used for health checks, updating calamari, and more.
    - [**Machine Proxies**](/docs/infrastructure/deployment-targets/proxy-support.md): allow you to specify a proxy server for Octopus to use when communicating with a Tentacle or SSH Target; you can also specify a proxy server when a Tentacle and the Octopus Server make web requests to other servers.
    - [**Accounts**](/docs/infrastructure/deployment-targets/index.md#accounts): credential details used during your deployments, including things like username/password, tokens, Azure and AWS credentials, and SSH key pairs.  

> - [**Projects**](/docs/projects/index.md): where the deployment process, configuration variables, runbooks are defined and configured to deploy and manage your software.

Reviewer: Slight change to the wording to make it work as introductory text.
```suggestion
## Projects

[**Projects**](/docs/projects/index.md) contain the deployment process, configuration variables, and runbooks to deploy and manage your software.
```

    - [**Release**](/docs/releases/index.md): a snapshot of the deployment process and the associated assets (packages, scripts, variables) as they existed when the release was created. 
    - [**Variables**](/docs/projects/variables/index.md): a value stored in the Octopus Server for use in different phases of your deployments.  Variables can be scoped to environments, steps, and more.  Variables allow you to have a consistent deployment process across your infrastructure without having to hard-code or manually update configuration settings that differ across environments, deployment targets, channels, or tenants.
    - [**Manual Interventions**](/docs/deployment-process/steps/manual-intervention-and-approvals.md): the approval step in Octopus Deploy.  Manual interventions can be scoped to specific teams and environments.
    - [**Runbooks**](/docs/runbooks/index.md): used to automate routine maintenance and emergency operations tasks like infrastructure provisioning, database management, and website failover and restoration.    

> - **Library** - stores items used across multiple projects

Reviewer: I've suggested artifacts and assets, does artifacts work in this context?
```suggestion
## Library 

The Octopus library stores artifacts and assets that can be used across multiple projects
```

Author: I'll update.  We use the term artifact as well (which I will add) in deployments.

The updated line will be:
> The Octopus library is where you store build artifacts and other assets that can be used across multiple projects.
